tests/emcute/Makefile

```
@@ -38,3 +37,8 @@ ethos:
 	$(Q)env -u CC -u CFLAGS $(MAKE) -C $(RIOTTOOLS)/ethos
 
 include $(RIOTBASE)/Makefile.include
+
+# Set CONFIG_EMCUTE_TOPIC_MAXLEN via CFLAGS if not being set via Kconfig.
+ifndef CONFIG_EMCUTE_TOPIC_MAXLEN
+  CFLAGS += -DCONFIG_EMCUTE_TOPIC_MAXLEN="249"   # 256 - 7
```

I believe this does not need quotes
```suggestion
  CFLAGS += -DCONFIG_EMCUTE_TOPIC_MAXLEN=249   # 256 - 7
```